app/src/main/res/values/strings.xml

```diff
@@ -5,24 +5,35 @@
   -->
 
 <resources xmlns:tools="http://schemas.android.com/tools">
+    <!-- This string is the full and formal title of the application, used in areas like a header or welcome screen -->
     <string name="app_name">Firefox Lockbox</string>
+    <!-- This is the short name or label of the application, used as the name in the Android app launcher -->
     <string name="app_label">Lockbox</string>
```

Comment:
I moved these two into a separate file in #250, as they're not supposed to be localized, per our conversation with Sandy.